tests/active/heuristic_test.py

```diff
@@ -307,5 +280,24 @@ def test_combine_heuristics_uncertainty_generator():
     ranks = heuristics(prediction_chunks)
     assert np.all(ranks == [1, 2, 0]), "Combine Heuristics is not right {}".format(ranks)
 
+def test_combine_heuristics_reorder_list():
```

Comment:
Add more test to ensure that the chunks works.

Reply:
what do you mean by chunks, dask kinda chunks or chunks for uncertainty_generator? 
Also what kinda test?

Reply:
Not sure why the coverage report doesn't pick it up, but I don't see tests for when heuristics is made of batches.